main/core/guards.ts

```diff
@@ -90,3 +91,11 @@ export function isInjectorLike(thing: any): thing is IInjector {
 export function isBoxedValue(thing: any): thing is IBoxedValue {
     return thing != null && thing.typeId === BOXED_TYPE_ID;
 }
+
+export function isDestroyable(thing: any): thing is IDestroyable {
```

Comment:
`thing`? usually we use `value` for this. (minor)

Reply:
> `thing`? usually we use `value` for this. (minor)

FIxed